lib/ViewModels/DataCatalogTabViewModel.js

```diff
@@ -38,4 +40,32 @@ DataCatalogTabViewModel.prototype.showInfo = function(item) {
     CatalogItemInfoViewModel.open('ui', item);
 };
 
+DataCatalogTabViewModel.prototype.clickEnabled = function(item) {
+    ga('send', 'event', 'dataSource', 'clickItem', item.name);
+
+    if (!item.isEnabled && defined(item.initialMessage) && item.initialMessage.length > 0) {
+        PopupMessageViewModel.open('ui', {
+            title: 'Data Set Information',
+            message: '<div>'+ item.initialMessage +'</div>'
+        });
+        item.initialMessage = undefined;
+    }
+
+    item.toggleEnabled();
+};
+
+DataCatalogTabViewModel.prototype.clickOpen = function(item) {
+    ga('send', 'event', 'dataSource', 'clickOpen', item.name);
+
+    if (!item.isOpen && defined(item.initialMessage) && item.initialMessage.length > 0) {
+        PopupMessageViewModel.open('ui', {
```

Comment:
I think it's fine.  If you're sending it to someone via a link it's not clear what should happen.  What if there are 10 sections open with a message each?